docs/features.md

```diff
@@ -30,6 +30,12 @@ The last line is tail prompt, which is optional. You can write no/single/multipl
 smile
 ```
 
+## Free-init
```

Comment:
FreeInit

Reply:
also:
- append this to https://github.com/continue-revolution/sd-webui-animatediff?tab=readme-ov-file#update "New features". 
- add your contribution to https://github.com/continue-revolution/sd-webui-animatediff?tab=readme-ov-file#thanks follow the same format

I will update the v2.0.0-a tag.

Reply:
done.
I also add an example in features.md.